### PR TITLE
chore: clean up synth.py

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: synth.py [hunk 1]
```diff
@@ -32,24 +32,20 @@
    version,
    generator_args={
      "grpc-service-config": f"grafeas/{version}/grafeas_grpc_service_config.json",
-     "package-name":f"@google-cloud/grafeas"
+     "package-name":"@google-cloud/grafeas"
      },
-     proto_path=f'/grafeas/{version}',
-     extra_proto_files=['google/cloud/common_resources.proto'],
```
Comment (alexander-fenster) [inline synth.py:38]: We don't need common resources here?

File: synth.py [hunk 2]
```diff
@@ -32,24 +32,20 @@
...
+     proto_path=f'/grafeas/{version}'
      )
  s.copy(library, excludes=[
    "README.md",
-   "package.json",
-   "src/v1/gapic-grafeas-v1.ts",
```
Comment (alexander-fenster) [inline synth.py:43]: Do you know why it was listed as excluded? Is there any problem with generation? A little bit confused here.

Comment (xiaozhenliu-gg5) [reply synth.py:43]: I checked the original conversion PR:
```
 s.copy(library, excludes=[
   "README.md",
   "package.json",
 -  "src/v1/grafeas_client.js",
 -  "test/gapic-v1.js",
 +  "src/v1/gapic-grafeas-v1.ts",
 +  "test/gapic-v1.ts",
   "protos/google/*"
 ])
```
I think for some reasons, the JS client lib skipped `src/v1/client` and `test/client.ts` so we did the same for TS client lib. But by removing these two files and rerun synthtool now, it works pretty well.

File: synth.py (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply synth.py:38]: This is not intended, thank you for catching this. There is one resource of `project`. 
But I removed the common_resources.proto and run synthtool, it works fine somehow.

**Pull request merged**